### DISCUSSION TITLE
No updates being found

### DISCUSSION THREAD

**Discussion by lgpage** [renovatebot/renovate#19919] [Request Help] [answered]
From the logs it looks like renovate is only finding an update for the codemirror package and no updates for any other packages.
The original update PR that renovate created for the codemirror package was closed and ignored, as it is a breaking change.

Can anyone explain why it is not finding updates for any other packages and/or what I can do to fix this?

Logs: https://app.renovatebot.com/dashboard#github/lgpage/rxtutor/978779166
Dashboard: https://github.com/lgpage/rxtutor/issues/174

**Comment by viceice** [comment, marked as answer]
by default renovate don't update ranges when the range satisfies the updated version.

- #19800

**Reply by lgpage** [reply]
Thank you @viceice 
I will read up on (and play around with) the rangeStrategy config option.
